### ISSUE TITLE
SONiC won't build on Debian 10 buster:  /bin/sh -c apt-get update needs quotes around arguments

### ISSUE THREAD

**Issue by ncalad** [Azure/sonic-buildimage#7424]
#### Steps to reproduce the issue:
1. clean checkout from git
2. make init
2. make configure PLATFORM=broadcom
3. make all

#### Describe the results you received:
The command '/bin/sh -c apt-get update &&            apt-get -y install               curl                         less                         perl                         procps                       python3                      python3-distutils            python3-pip                  rsyslog                      vim-tiny                     libatomic1                   libjemalloc2                 liblua5.1-0                  lua-bitop                    lua-cjson                    libdaemon0                   libdbus-1-3                  libjansson4                  iproute2                     net-tools                    jq                           libzmq5' returned a non-zero code: 100

#### Describe the results you expected:
I expected the build to succeed

#### Output of `show version`:
n/a
```
(paste your output here)
```
#### Output of `show techsupport`:
n/a

```
(paste your output here or download and attach the file here )
```

**Comment by lguohan** [comment]
from daily build here, it looks fine. https://dev.azure.com/mssonic/build/_build?definitionId=138

**Comment by ncalad** [comment]
I have been trying unsuccessfully to build SONiC for a couple of weeks now. I started with ubuntu and ran into python version issues there, that's why I switched to Debian. 

Today, I set up a fresh Debian 10 (buster) instance on Amazon EC2. After installing docker and installing the prerequisites, python, pip, jinja2 and j2cli and adding the user to the docker group, I can now run make configure PLATFORM=broadcom. 

Now when I run make all, it has a problem accessing /proc/modules.  I believe the original problem has gone because it is failing later in the build process. The following error repeats for each test: 


name = 'eeprom'

    def isModuleLoaded(name):
>      with open('/proc/modules') as f:
E      IOError: [Errno 2] No such file or directory: '/proc/modules'
